src/python/create.py

```diff
@@ -26,13 +28,22 @@ def create_experiments(parsed_experiments, collection):
                 # Only attribute types should be in Experiment
                 if cell_type == "attribute":
```

Comment:
I suggest to use switch/case or else if.  Why testing cell_type against "identifier" when we know above its value is "attribute"?